src/app.ts

```diff
@@ -38,12 +39,9 @@ app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "html");
 
 // apply middleware
-// TODO get key and timeout from config
 app.use(cookieSessionMiddleware({
-  keys: ["TO BE CHANGED"],
-  // Cookie Options
-  maxAge: 60 * 60 * 1000 // 1 hours
-} as any) as any);
+  keys: [COOKIE_SECRET]
```

Comment:
should we leave the maxAge in until we add it to config?

Reply:
Max age isnt used by the middleware as far as i can see